src/models/architectures/cnns/architectures/competitive_multiscale_cnn.py

```diff
@@ -0,0 +1,74 @@
+import torch
+import torch.nn as nn
+
+
+class MaxOut(nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    def forward(self, inputs):
+        cur_max = inputs[0]
+        for inp in inputs[1:]:
+            cur_max = torch.max(cur_max, inp)
```

Comment:
Are you sure that this function does not take Sequential object (like inputs) as an argument (you search max elementwise)? I'm curious and it is hard to read it from documentation.

Reply:
Nope, it takes two Tensors and compares them elementwise:

Documentation says:

> `torch.max(input, other, out=None) → Tensor`
Each element of the tensor input is compared with the corresponding element of the tensor other and an element-wise max is taken.
The shapes of input and other don’t need to match, but they must be broadcastable.